src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -90,7 +90,7 @@ public void Default(EnvironmentVariableTarget target)
         [InlineData(EnvironmentVariableTarget.User)]
         public void ModifyEnvironmentVariable(EnvironmentVariableTarget target)
         {
-            const string varName = "Test_ModifyEnvironmentVariable";
+            string varName = $"Test_ModifyEnvironmentVariable ({target})";
```

Comment:
I didn't realize until now that env vars could hvae spaces in the name, but clearly they can..